src/Userenv.Desktop/Userenv.cs

```diff
@@ -15,7 +15,7 @@ namespace PInvoke
     public static partial class Userenv
     {
         /// <summary>
-        /// Retrieves the environment variables for the specified user. This block can then be passed to the <see cref="Kernel32.CreateProcessAsUser(IntPtr, string, string, SECURITY_ATTRIBUTES*, SECURITY_ATTRIBUTES*, bool, CreateProcessFlags, void*, string, ref STARTUPINFO, out PROCESS_INFORMATION)"/> function.
+        /// Retrieves the environment variables for the specified user. This block can then be passed to the CreateProcessAsUser function.
```

Comment:
Please restore the `<see cref` tag, adding parameter types to resolve the compiler warning if necessary.